FactoryWeather/App/VIP Home/Views/BodyView.swift

```diff
@@ -39,7 +39,7 @@ class BodyView: UIView {
 }
 
 extension BodyView {
```

Comment:
The `BodyView` should have a `ViewModel` typealias that contains only the data relevant to the `BodyView` and the `updateProperties` modified to work with the `BodyView.ViewModel`.